.editorconfig

```diff
@@ -2,7 +2,6 @@ root = true
 
 [*]
 indent_size = 2
-end_of_line = lf
```

Comment:
Why remove this?

Reply:
Eol should be handled by `.gitattributes`.

Reply:
I'm not sure an IDE would read that file, though. Although I'm not sure whether _any_ idea actually uses CRLF by default anymore.

Reply:
I test it in windows, make a new `Test.kt` with CRLF by default.